ruby/ql/lib/codeql/ruby/frameworks/core/Array.qll

```diff
@@ -2052,18 +2062,23 @@
     }
   }
 
+  private class InjectMethodName extends string {
+    // `reduce` is an alias for `inject`.
+    InjectMethodName() { this = ["inject", "reduce"] }
+  }
+
   abstract private class InjectSummary extends SummarizedCallable {
     MethodCall mc;
+    InjectMethodName methodName; // adding this as a field helps give a better join order
```

Comment:
Either remove comment, or add to all fields (e.g. the one in `GrepSummary` doesn't have the comment).

Reply:
OK, I've added the comment to all the fields.